packages/side-nav/test/side-nav-item.test.js

```diff
@@ -94,6 +94,23 @@ describe('side-nav-item', () => {
     });
   });
 
+  describe('path alias', () => {
+    beforeEach(async () => {
+      item = fixtureSync(`<vaadin-side-nav-item path="/another-path" pathAliases="/"></vaadin-side-nav-item>`);
```

Comment:
I would be good to have a test where `pathAliases` contains several aliases to improve test coverage.

Reply:
I added a few more tests, which includes one where the item has multiple aliases in `pathAliases `.